Automattic/jetpack
# Add Jetpack Connection to Launchpad

This PR adds a Jetpack Connection screen and placeholder admin page to the standalone Launchpad plugin.

After activating the plugin:

![image](https://user-images.githubusercontent.com/1713699/145054215-79f2b498-2031-458b-a22a-5a0181dfb11e.png)

Setting up the connection:

![image](https://user-images.githubusercontent.com/1713699/145054232-0674ad26-cd62-475b-b708-3b3b864679d2.png)

Admin page:

![image](https://user-images.githubusercontent.com/1713699/145054251-013be16f-38e2-4ce4-8d55-675794f7e0f7.png)


#### Changes proposed in this Pull Request:

* Add the ability to connect to Jetpack from the plugin.
* Add placeholder admin page.

#### Jetpack product discussion
p9dueE-3UC-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:






* Build the plugin: `jetpack build plugins/launchpad`.
* Activate the plugin.
* Confirm that you can connect to Jetpack (this doesn't work on a local install).
* Confirm that the admin screen (`Jetpack > Launchpad` in the admin menu) shows the connection status.
